.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
Yes if it's an option that can be disabled locally. In other words: I'm fine with Travis hitting mocks, but tests need to be able to hit the live API when debugging locally. 

Would love to see this land. Hopefully that change can push it over the finish line.